test/jdk/jdk/incubator/vector/VectorReshapeTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The Copyright should be " Copyright (c) 2021," since it's a new file added in 2021.

Reply:
> The Copyright should be " Copyright (c) 2021," since it's a new file added in 2021.

done